client/modules/_hooks/src/workspace/useGetJobs.ts

```diff
@@ -38,6 +38,23 @@ export function useGetJobs(
   return useQuery(getJobsQuery(operation, queryParams));
 }
 
+const getJobQuery = (
+  operation: TJobGetOperations,
+  queryParams: TJobParamsType
+) => ({
+  queryKey: ['workspace', 'getJobs', operation, queryParams],
```

Comment:
Should the query key be for `getJob`?
```suggestion
  queryKey: ['workspace', 'getJob', operation, queryParams],
```

Reply:
good catch, will fix